### PR TITLE
Software list: "do not change" label.

### PR DESCRIPTION
**Pull request by Darleev**
# Description
Software list improvment. Added additional line with strong recommendation.

#### Related issue:
Internal issue.

## Check list
- [+] Documentation is updated (if applicable)

### CODE REVIEW

File: images/linux/scripts/installers/1604/preparemetadata.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 source $HELPER_SCRIPTS/document.sh
 
+WriteItem "<!--- This file is autogenerated, it is strongly recommended not to modify it. -->"
```
Comment (alepauly) [inline images/linux/scripts/installers/1604/preparemetadata.sh:10]: I think this message can be more direct, something like:
```
DO NOT EDIT - This markdown file is autogenerated
```

Comment (Darleev) [reply images/linux/scripts/installers/1604/preparemetadata.sh]: I have added the string according to provided example.

**Pull request merged**